lib/runner.js

```diff
@@ -277,7 +279,7 @@ internals.executeExperiments = async function (experiments, state, skip) {
 
         // Sub-experiments
 
-        await internals.executeExperiments(experiment.experiments, state, skipExperiment);
+        await internals.executeExperiments(experiment.experiments, state, skipExperiment, state.currentContext);
```

Comment:
Since this is attached to `state`, and `state` is already an argument, do you need to add a new argument?

Reply:
state gets reused, so I want to avoid any confusion